server/build/gzip.js

```diff
@@ -8,7 +8,8 @@ export default async function gzipAssets (dir) {
 
   const coreAssets = [
     path.join(nextDir, 'commons.js'),
-    path.join(nextDir, 'main.js')
+    path.join(nextDir, 'main.js'),
+    path.join(nextDir, 'static-bundle.js')
```

Comment:
*I am not familiar with webpack to judge this, so I will only put my concern here.*

Can't `static-bundle.js` get merged with `commons.js` and `main.js`? It looks like all three of these are included on every page anyway, so splitting them up makes 3 separate requests.